content/en/docs/concepts/containers/runtime-class.md

```diff
@@ -170,8 +164,6 @@ Nodes](/docs/concepts/scheduling-eviction/assign-pod-node/).
 
 You can specify _overhead_ resources that are associated with running a Pod. Declaring overhead allows
 the cluster (including the scheduler) to account for it when making decisions about Pods and resources.  
-To use Pod overhead, you must have the PodOverhead [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
```

Comment:
PodOverhead is still beta, right?

Reply:
Oh, right. I will restore this sentence.